lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1512,6 +1521,27 @@ OpFoldResult Aten__Getitem__DictStrOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// Aten__Contains__IntListOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult Aten__Contains__IntListOp::fold(ArrayRef<Attribute> operands) {
+  auto arrayConstruct = getListConstructIfNotModified(l());
+  if (!arrayConstruct)
+    return nullptr;
+
+  auto targetKey = item();
```

Comment:
nit: the convention in this repo is to name variables after their argument name in pytorch. So this should be named `item`